script/build.sh

```diff
@@ -27,35 +30,46 @@ BUILDS="$WORKINGDIR/interpolation/builds";
 [ -d $BUILDS ] || mkdir -p $BUILDS;
 
 # a directory where this specific build will live
-export BUILDDIR="$BUILDS/$TIMESTAMP";
+export BUILDDIR="${BUILDDIR:-"$BUILDS/$TIMESTAMP"}";
 [ -d $BUILDDIR ] || mkdir -p $BUILDDIR;
 
 # location of temp files
 export SQLITE_TMPDIR="$BUILDDIR/tmp"; # a directory with enough free space to store sqlite tmp files
 export PBF2JSON_TMPDIR="$BUILDDIR/tmp/leveldb"; # a directory with enough free space to store leveldb tmp files
 
+echo "$BUILDDIR"
+echo "$WORKINGDIR"
```

Comment:
These lines should have some sort of text explaining what is printed before they print. Or if they were just for debugging maybe they can be removed.